### PR TITLE
Photo upload

### PR DESCRIPTION
**Pull request by PearlynnT**
# Changes by Pearlynn
## Summary
- Added Cloudinary Widget to upload photos
- Refactor code and created a form partial
- Added dropdown for photo type
- Added validation checks if input fields are present
- Added file format message

## Need help in
- Loading `cloundinary_upload.js` into `application.js`
- Linking photos to projects/home page

### To note:
There are currently two columns for photo url in the Photos table

# Changes by Kevin
## Summary
- updated photo schema and allowed for user to select viewing in home page + specifically a project page
- photo table has no redundancy - redundant information like type and extra url field removed
- added dropdown to select project and checkbox to select if photo should be displayed in home page
- added seed data for locations
- added cloudinary credentials to secrets file, making .env files unnecessary

### CODE REVIEW

**Comment by ryanpavlik24** [comment]
I think this should point to dev not main right?

**Comment by xkjjx** [comment]
> I think this should point to dev not main right?

Yes - I just changed it. Also I need to incorporate the environment variables into Ruby's solution.

**Comment by xkjjx** [comment]
@PearlynnT credentials.yml.enc needs to be tracked in order for our program to work
The reason it is ok to be tracked is because it is encrypted. In order for the program to work, it also needs access to the master.key file which contains the key needed to unencrypt the file.

**Comment by xkjjx** [comment]
@PearlynnT @ryanpavlik24 all base features are done - I think we just need to add tests - let me know what you guys think.

**Comment by ryanpavlik24** [comment]
Is this good to be merged? @PearlynnT

**Comment by xkjjx** [comment]
@PearlynnT @ryanpavlik24 Let me know what you guys think.

**Review by xkjjx — APPROVED** [review]
This looks good to me.

File: app/views/photos/edit.html.erb [hunk 1]
```diff
@@ -1,41 +1,7 @@
...
-                    <% end %>
-                </ul>
-            </div>
-        <% end %>
 
-    <div class="field">
-        <%= form.label :Photo_Link %>
-        <%= form.text_field :photoLink %>
-    </div>
+<p> <%= @photo.url %> </p>
```
Comment (PearlynnT) [inline app/views/photos/edit.html.erb:3]: `<p> <a href="<%= photo.url %>" target="_blank"></a></p>`
maybe change photo url to a hyperlink instead

**Review by PearlynnT — COMMENTED** [review]
Looks good to merge, thanks!!

File: app/views/photos/edit.html.erb (hunk 1 above)
Comment (xkjjx) [reply app/views/photos/edit.html.erb]: Done.

**Pull request merged**